src/wp-includes/rest-api/endpoints/class-wp-rest-settings-controller.php

```diff
@@ -237,6 +237,7 @@ protected function get_registered_options() {
 
 			$default_schema = array(
 				'type'        => empty( $args['type'] ) ? null : $args['type'],
+				'label'       => empty( $args['label'] ) ? '' : $args['label'],
```

Comment:
@TimothyBJacobs, in Gutenberg shim, we went with the `title` key for the schema.  Would it make sense to have a direct mapping to the argument here, or is the `title` reserved key for similar data in schemas?

Reply:
Yeah it should be `title` as the prop name. That is how it's defined in the JSON Schema spec.

Reply:
Thanks for the clarification!